lib/Default/SmrPlayer.class.inc

```diff
@@ -465,16 +465,17 @@ class SmrPlayer extends AbstractSmrPlayer {
 		$this->hasChanged=true;
 	}
 
-	function getAttackColour() {
```

Comment:
Could you readd these or remove them completely please (I know these functions are not used, but if we're gonna have the db fields/class properties then it would be nice to keep the methods available to make use of, or just get rid of it altogether).  For reference these were a 1.5 thing where you could choose the colour of the flashing screen when you get attacked

Reply:
We already have one vote for keeping it (and even re-implementing the ability to customize the color), so unless you object I'll keep that code in there.

Reply:
Keeping it works for me